tests/providers/apache/kafka/hooks/test_admin_client.py

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import logging
+import unittest
+
+import pytest
+
+from airflow import AirflowException
+
+# Import Hook
+from airflow.providers.apache.kafka.hooks.admin_client import KafkaAdminClientHook
+
+log = logging.getLogger(__name__)
+
+
+class TestSampleHook(unittest.TestCase):
```

Comment:
Net-new tests should use `pytest` instead.

Reply:
i don't know what you mean. All of these tests are pytest compatible.

Reply:
Relatedly: https://github.com/apache/airflow/issues/29305. Tests shouldn't be unittest cases.